### ISSUE TITLE
Deep linking from notification

### ISSUE THREAD

**Issue by docherty** [katzer/cordova-plugin-local-notifications#390]
I'm using Ionic with this plugin and it works great on iOS and Android (thank you devs). In Ionic I can deep link within an app by setting the URL. So, I'd like to modify the local notification created via this plugin to customise the post-click (or "slide to view") destination. For example, when a user clicks a notification, rather than it simply opening the app it opens the app on a specific url ( e.g. "You have a new message" --> "/#/inbox/msg-27").

I'm sorry if it's in the docs but I have had a good search but didn't find anything. If anyone can help I'd be very grateful.

Many thanks,
James

**Comment by ds8k** [comment]
This is more of an Ionic question than an issue.

I'm working on an app built with Sencha touch. I'm currently using the onclick function to modify the webview's address:

```
window.cordova.plugins.notification.local.onclick = function () {
                     window.location.href = url;
            }
```

This is only triggered if the app is still running in the background.

**Comment by docherty** [comment]
Thanks for the info @ds8k - I realised it was a question rather than an issue but didn't know how to tag it as a question (sorry).

Your onclick solution makes sense and seems like it should work. In my use case, the notifications will only get created if the app is running in the background so I should be fine. Thank you for the suggestion.

On a related note, I was looking at an old xcode project where I had customised the notification and noticed an alertAction property which allowed customisation of the "slide to view" message on iOS. Is this an option available in this plugin?
